features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+namespace RoaveTest\BehatPsrContainer;
+
+use Behat\Behat\Context\Context;
+
+final class FeatureContext implements Context
+{
+    /**
+     * @var TestService
+     */
+    private $testService;
+
+    public function __construct(TestService $testService)
+    {
+        $this->testService = $testService;
+    }
+
+    /**
+     * @Given /^I have a Zend\\ServiceManager container$/
+     */
+    public function iHaveAZendServiceManagerContainer()
+    {
```

Comment:
An assertion should be added here, verifying that the container can be injected too